kyaml/yaml/alias.go

```diff
@@ -21,12 +22,23 @@ type Style = yaml.Style
 type TypeError = yaml.TypeError
 type Unmarshaler = yaml.Unmarshaler
 
-var Marshal = yaml.Marshal
+var Marshal = func (in interface{}) (out []byte, err error) {
+	var buf bytes.Buffer
+	e := NewEncoder(&buf)
+	e.SetIndent(2)
```

Comment:
Noting that this is a change - Marshal currently indents with 4 spaces.

Here if we use e.SetIndent(4), sequence elements still only get 2 while other elements get 4.